lib/remote-settings.js

```diff
@@ -7,15 +7,20 @@ const AppConstants = require("../app-constants");
 const BREACHES_COLLECTION = "fxmonitor-breaches";
 const FX_RS_COLLECTION = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-workspace/collections/${BREACHES_COLLECTION}`;
 const FX_RS_RECORDS = `${FX_RS_COLLECTION}/records`;
-const FX_RS_BEARER_TOKEN = AppConstants.FX_REMOTE_SETTINGS_BEARER_TOKEN;
+const FX_RS_WRITER_USER = AppConstants.FX_REMOTE_SETTINGS_WRITER_USER;
+const FX_RS_WRITER_PASS = AppConstants.FX_REMOTE_SETTINGS_WRITER_PASS;
+const FX_RS_WRITER_AUTH = Buffer.from(`${FX_RS_WRITER_USER}:${FX_RS_WRITER_PASS}`).toString("base64");
 
 const RemoteSettings = {
 
   async whichBreachesAreNotInRemoteSettingsYet(breaches) {
+    console.log("whichBreachesAreNotInRemoteSettingsYet");
+    console.log("FX_RS_RECORDS: ", FX_RS_RECORDS);
+    console.log("FX_RS_WRITER_AUTH: ", FX_RS_WRITER_AUTH);
```

Comment:
LGTM. Not sure if we meant to leave in the `console.log()` statements, or if those were meant for debugging. /shrug

Reply:
Nope, I'll take them out when I'm done debugging on remote settings stage server.